funcbench/manifests/benchmark/2_job.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: funcbench-test-{{ .PR_NUMBER }}
+  namespace: funcbench-{{ .PR_NUMBER }}
+spec:
+  # never re-create a new pod
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: funcbench
+        image: nevill/funcbench:20200324
+        imagePullPolicy: Always
+        args:
+          - "-v"
```

Comment:
I think passing the full flag names for funcbench here will improve readability.

Reply:
also please change the name of the image to  ` docker.io/prominfra/funcbench:master` from `nevill/funcbench:20200324`

Reply:
Probably I need another PR to change the image name ? Otherwise you cannot really test my workflow. Because `prominfra/funcbench:master` is not sync with this PR. When `prombench` executes the manifest, it cannot find the appropriate docker image.

Reply:
okay we keep it this way for testing now then, just a reminder to change it to the correct name later in the same PR before merging.